src/scripts.js

```diff
@@ -69,11 +69,11 @@ function getUsersRecentDate(id, dataSet) {
 }
 
 //************remove after getting rid of in other functionality
-function makeRandomDate(userStorage, id, dataSet) {
-  // gives a random date (used where?)
-  var sortedArray = userStorage.makeSortedUserArray(id, dataSet); // sorts data set by date from recent
-  return sortedArray[Math.floor(Math.random() * sortedArray.length + 1)].date;
-}
+// function makeRandomDate(userStorage, id, dataSet) {
+//   // gives a random date (used where?)
+//   var sortedArray = userStorage.makeSortedUserArray(id, dataSet); // sorts data set by date from recent
+//   return sortedArray[Math.floor(Math.random() * sortedArray.length + 1)].date;
+// }
```

Comment:
Why did we comment this out

Reply:
We are removing the random week from the page as it is not used in a user friendly way